ksql-engine/src/main/java/io/confluent/ksql/codegen/CodeGenRunner.java

```diff
@@ -57,6 +57,13 @@
   private final FunctionRegistry functionRegistry;
   private final ExpressionTypeManager expressionTypeManager;
 
+  public static final String[] CODEGEN_IMPORTS = new String[]{
```

Comment:
This should be an immutable list...

Reply:
Yes, it was failing in Jenkins too. Changed it into ImmutableList.